addons/default-storage-class/storage-class.yaml

```diff
@@ -124,3 +124,26 @@ provisioner: csi.hetzner.cloud
 volumeBindingMode: WaitForFirstConsumer
 allowVolumeExpansion: true
 {{ end }}
+
+{{ if eq .Cluster.CloudProviderName "nutanix" }}
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: ntnx-csi
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: csi.nutanix.com
+parameters:
+  storageType: NutanixVolumes
+  csi.storage.k8s.io/provisioner-secret-name: ntnx-secret
+  csi.storage.k8s.io/provisioner-secret-namespace: kube-system
+  csi.storage.k8s.io/node-publish-secret-name: ntnx-secret
+  csi.storage.k8s.io/node-publish-secret-namespace: kube-system
+  csi.storage.k8s.io/controller-expand-secret-name: ntnx-secret
+  csi.storage.k8s.io/controller-expand-secret-namespace: kube-system
+  storageContainer: {{ default "Default" .Params.storageContainer | quote }}
```

Comment:
Can a user even change this?

Reply:
No this is a copy paste leftover from kubeone. I thought i removed all